db/fixtures/miq_product_features.yml

```diff
@@ -5108,6 +5108,92 @@
           :description: Edit Job Templates Tags
           :feature_type: control
           :identifier: configuration_script_tag
+# Ansible Tower
+- :name: Ansible Tower
+  :description: Everything under Ansible Tower
+  :feature_type: node
+  :identifier: ansible_tower
```

Comment:
I am really against us hardcoding this as ansible_tower.  It should be reflected as AutomationManager, or, perhaps ExternalAutomationManager.  The UI should never code to a specific provider but instead to the abstraction layer above it. cc @dclarizio @chessbyte